### PR TITLE
Implement PointPen behaviour for BezierPath()

### PR DESCRIPTION
**Pull request by jenskutilek**
Per the DrawBot docs, a BezierPath can be [used like a PointPen](https://www.drawbot.com/content/shapes/bezierPath.html#drawBot.context.baseContext.BezierPath.beginPath). I tried to implement it and it seems to work, but I'm not sure that's all there's to it ;)

### CODE REVIEW

**Comment by justvanrossum** [comment]
Hm, I don't like the multiple inheritance and I don't like the circular reference that never gets broken. Let me think.

**Comment by justvanrossum** [comment]
Have a look at what DB does:

https://github.com/typemytype/drawbot/blob/f96bb7fb26ec2163fa5400ee2c38676b7af6f6bb/drawBot/context/baseContext.py#L221-L227

It uses a temporary PointToSegmentPen that gets removed at `endPath()`.

**Comment by jenskutilek** [comment]
Of course! I should have checked the drawBot code in the first place.

File: src/drawbot_skia/path.py [hunk 1]
```diff
@@ -51,8 +51,28 @@ def _qCurveToOne(self, pt1, pt2):
...
+        self._pointToSegmentPen.addPoint(
+            point,
+            segmentType=segmentType,
+            smooth=smooth,
+            name=name,
+            identifier=identifier,
+            **kwargs
+        )
+
     def _endPath(self):
```
Comment (justvanrossum) [inline src/drawbot_skia/path.py:70]: This needs to become `endPath(self):`
```suggestion
    def endPath(self):
```

**Review by justvanrossum — CHANGES_REQUESTED** [review]
Thanks! Apart from one nit, this looks good.

**Review by justvanrossum — APPROVED** [review]

**Pull request merged**